docs/source/release/v6.0.0/diffraction.rst

```diff
@@ -35,6 +35,9 @@ New features
 Improvements
 ############
 - :ref:`WANDPowderReduction <algm-WANDPowderReduction>` now accepts a sequence of input workspaces, combining them to reduce to a single spectrum.
+- The height of the :ref:`func-BackToBackExponential` peak is now preserved when changing the FWHM sliders when fitting.
+- Modify filenames of xye outputs from running a focus in the Pearl power diffraction scripts
```

Comment:
These two lines (39 and 40) look like they've become duplicated in the merge. They also exist higher up in this file. I think they can be deleted here